src/DeserializationPojo/ListResourceDataPojo.java

```diff
@@ -0,0 +1,43 @@
+package DeserializationPojo;
+
+public class ListResourceDataPojo {
```

Comment:
Better name `ResourcePojo`. The term Data isn't adding any information. The fact that this object is used in the list resource api isn't important. Rather using api info in class name can become misleading if you have multiple apis that return the same object. So it's useful to name it based on what it is rather than where it is used.